sign/sign.go

```diff
@@ -139,11 +139,6 @@ func (s *Signer) SignImage(reference string) (*SignedObject, error) {
 		return nil, errors.Wrapf(err, "sign reference: %s", reference)
 	}
 
-	if !s.options.AttachSignature {
```

Comment:
not sure if we need this and why we have that before

Reply:
For testing? It has been added in this patch: https://github.com/kubernetes-sigs/release-sdk/pull/25/files#diff-9c6c93e3b9d57b2aacb32e9b81617b1d8e41614365745f2e1845f23120605b23R94